test/service/test_pagination.py

```diff
@@ -129,14 +130,16 @@ def test_search_after_last_page(self):
         Tests that the last page returned in search_after pagination mode is correct.
         :return:
         """
-        content = requests.get("{}?sort=entryId&order=asc".format(self.get_base_url())).content
+        content = requests.get(self.get_base_url() + '?sort=entryId&order=asc').content
         json_response = json.loads(content)
         self.assert_page1_correct(json_response)
         # Store the search_after for the last result of the first page.
         search_after_lrfp = json_response['pagination']['search_after']
         search_after_lrfp_uid = json_response['pagination']['search_after_uid']
-        content = requests.get("{}?sort=entryId&order=asc&search_after={}&search_after_uid={}"
-                               .format(self.get_base_url(), search_after_lrfp, search_after_lrfp_uid)).content
+        content = requests.get(
```

Comment:
Let's not mix f strings and concatenation. Since you're within balanced constructs, you can split the string literals. There already is a code section like that a few lines down which sets the precedent.